src/plugins/privacy/activitystream/activitystream.php

```diff
@@ -10,6 +10,8 @@
 
 // No direct access.
 defined('_JEXEC') or die();
+use Joomla\CMS\User\User;
+use Joomla\CMS\Table\User;
```

Comment:
potential class name conflict

Namespaces: Classes under different namespaces paths with same names - Use alias and use: eg:
```
use Joomla\CMS\User\User;
use Joomla\CMS\Table\User as UserTable;

$userTable = UserTable::getTable();
```